### PR TITLE
Select option key duplication fix

### PR DESCRIPTION
**Pull request by ozerovs**

### CODE REVIEW

File: src/select/index.es [hunk 1]
```diff
@@ -54,7 +54,7 @@ class Select extends InputComponent {
 
   renderOption(item, index) {
     const { key } = getOption(item);
-    return <option value={index} key={key}>{key}</option>;
+    return <option value={index} key={index + key}>{key}</option>;
```
Comment (suprMax) [inline src/select/index.es:57]: why is this necessary? do we have the same text option several times? how does that work?

Comment (ozerovs) [reply src/select/index.es:57]: image

In this example we have businesses with same name(key in here) but with different IDs. 
So all is good till the moment we are rendering them as list, and they are receiving keys from the name prop. 
And passing other keys to Select won't really work since Select is using key prop from here to render options in select element, as far as it seems.
